youtube_dl/extractor/ceskatelevize.py

```diff
@@ -201,40 +201,15 @@ def _real_extract(self, url):
         return self.playlist_result(entries, playlist_id, playlist_title, playlist_description)
 
     def _get_subtitles(self, episode_id, subs):
-        original_subtitles = self._download_webpage(
+        vtt_subs = self._download_webpage(
             subs[0]['url'], episode_id, 'Downloading subtitles')
-        srt_subs = self._fix_subtitles(original_subtitles)
         return {
             'cs': [{
-                'ext': 'srt',
-                'data': srt_subs,
+                'ext': 'vtt',
+                'data': vtt_subs,
```

Comment:
If it does not require pre-processing anymore then it should be extracted as url not data.